public/index.html

```diff
@@ -9,9 +9,16 @@
       name="description"
       content="FueledUp is an e-commerce platform for all things Formula1" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!-- MockMan API styles -->
+
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/mockman-js@latest/dist/style.css" />
 
     <!-- PowertrainUI library -->
-    <link rel="stylesheet" href="https://powertrain-ui.netlify.app/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://powertrain-ui.netlify.app/styles.css" />
```

Comment:
Change to below for better performance

```suggestion
    <link
      rel="preconnect"
      href="https://powertrain-ui.netlify.app" />
    <link
      rel="stylesheet"
      href="https://powertrain-ui.netlify.app/styles.css" />
```

Reply:
Did not know about preconnect, got to learn tis advantages and how it works.
Added the changes!